mnefun/_mnefun.py

```diff
@@ -1567,7 +1616,7 @@ def gen_covariances(p, subjects, run_indices):
             epochs = Epochs(raw, events, event_id=None, tmin=p.bmin,
                             tmax=p.bmax, baseline=(None, None), proj=False,
                             reject=use_reject, flat=use_flat, preload=True)
-            epochs.pick_types(meg=True, eeg=True, exclude=[])
+            epochs.pick_types(meg=True, eeg=True, exclude=())
```

Comment:
It won't make much of a difference. I slightly prefer tuples for this sort of thing since they are immutable, but I'm not usually very careful about it. Not really sure why I changed this one...